src/sass/style.scss

```diff
@@ -40,7 +40,24 @@ html {
     object-fit: cover;
     width: 100vw;
 }
-
+.banner-text{
+    position: absolute;
+    bottom: 25rem;
+    left: 20rem;
+    background-color: white;
+    width: 25rem;
+    height: 10rem;
+    border: 0.5rem solid lightgray;
+}
+.text{
+    text-align: center;
+}
+.banner-text-main, {
+    font-size: 1.5rem;
+    padding-top:3rem;
+    font-weight: bold;
```

Comment:
Remember not to use bold, use 700 (look for faux web typography and Zach Letherman).

Reply:
Will do. Thanks.

Reply:
I'll look this up. Thanks